app/src/main/java/com/einmalfel/podlisten/PodListenApp.java

```diff
@@ -42,7 +42,7 @@ public static Context getContext() {
 
   @Override
   public void onCreate() {
-    instance = this;
+    setInstance(this);
```

Comment:
same here: it's a bug that "instance" is not volatile, but what is the reason for non-synchronized setter?